src/src/components/Login.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div></div>
+</template>
+
+<script>
+import { defineComponent } from "@vue/runtime-core";
+
+export default defineComponent({
+    name: 'Login',
+    setup(){
+        return {};
+    },
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
  },
```